schema/meta_schema_types.yaml

```diff
@@ -1264,3 +1266,48 @@
     - enumValue: WARN4
       description: WARN4, severity number 16.
   isSdkExtensionPlugin: false
+- type: SpanKind
+  enumValues:
+    - enumValue: CLIENT
+      description: CLIENT, a client span.
+    - enumValue: CONSUMER
+      description: CONSUMER, a consumer span.
+    - enumValue: INTERNAL
+      description: INTERNAL, an internal span.
+    - enumValue: PRODUCER
+      description: PRODUCER, a producer span.
+    - enumValue: SERVER
+      description: SERVER, a server span.
+  isSdkExtensionPlugin: false
+- type: ExperimentalComposableRuleBasedSampler
+  properties:
+    - property: rules
+      description: The rules for the sampler, matched in order. Each rule can have multiple match conditions - the sampler will be applied if all match. If no conditions are specified, the rule matches all spans that reach it.
+  isSdkExtensionPlugin: false
+- type: ExperimentalComposableRuleBasedSamplerRule
+  properties:
+    - property: attribute_patterns
+      description: Patterns to match against a single attribute.
+    - property: attribute_values
+      description: Values to match against a single attribute.
+    - property: is_root
```

Comment:
Would be cool to be able to differentiate between the trace root and the root span within a service's boundaries (sometimes called the local root).